### PR TITLE
pgSQL/DB set up

### PR DESCRIPTION
**Pull request by sarapowers**
Installed `node-postgres` and `pg-promise` to utilize Pool and Promise functionality for our pgSQL database. 
Created **_models_** folder containing **_db.js_**,  **_userModel.js_**, and **_favsModel.js_**. 
**_db.js_** contains functionality for creating our pgSQL Pool and exporting its query function to other files. 
**_userModel.js_** contains the Schema for our Users table in our databse.

Created **_controllers_** folder containing **_dbController.js_** that contains an object of query strings for easy implementation and repeated functionality in our database functions.

### CODE REVIEW

**Comment by mitchelsevere** [comment]
Wonderful!

File: server/models/db.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { Pool } = require('pg');
+
+//URI to sql database hosted on ElephantSQL
```
Comment (alexbednarek4) [inline server/models/db.js]: Love the use of pool here, Sara!

**Comment by natattackvick** [comment]
Awesome job, Sara! Clean db file!

File: server/models/db.js (hunk 1 above)
Comment (sarapowers) [reply server/models/db.js]: 🏊🏻‍♀️🌊

Comment (natattackvick) [reply server/models/db.js]: Keep ridin the data waves, Sara!

Comment (mitchelsevere) [reply server/models/db.js]: Wow! Now that's a pool.

Comment (nabramow) [reply server/models/db.js]: 🐠

**Pull request merged**